TaskManager/Program.cs

```diff
@@ -18,16 +18,16 @@ public static async Task Main()
             string command;
             do
             {
-                Console.WriteLine("0. Wyświetl użytkowników");
-                Console.WriteLine("1. Dodaj zadanie");
-                Console.WriteLine("2. Usuń zadanie");
-                Console.WriteLine("3. Pokaż szczegóły zadania");
-                Console.WriteLine("4. Wyświetl wszystkie zadania");
-                Console.WriteLine("5. Wyświetl zadania wg statusu");
-                Console.WriteLine("6. Szukaj zadania");
-                Console.WriteLine("7. Zmień status zadania");
-                Console.WriteLine("8. Przypisz zadanie");
-                Console.WriteLine("9. Zakończ");
+                Console.WriteLine("0. Print users");
```

Comment:
Proponuję aby to tłumaczenie było: `Show users`
` Console.WriteLine("0. Show users");`

Reply:
jestem za !!

Reply:
Ok poprawię to